src/main/scala/io/viash/config/Config.scala

```diff
@@ -87,7 +89,30 @@ case class Config(
   engines: List[Engine] = Nil,
 
   @undocumented
-  info: Option[Info] = None
+  info: Option[Info] = None,
+
+  @description("The project config content used during build.")
+  @since("Viash 0.9.0")
+  @undocumented
+  project_config: Option[ViashProject] = None,
```

Comment:
I'm wondering. Should we then also rename `ViashProject` to `ProjectConfig`? Or leave as is?

Reply:
Makes sense. Done!